test/node-api/test_buffer/test_buffer.c

```diff
@@ -140,6 +189,7 @@ static napi_value Init(napi_env env, napi_value exports) {
       DECLARE_NODE_API_PROPERTY("bufferInfo", bufferInfo),
       DECLARE_NODE_API_PROPERTY("staticBuffer", staticBuffer),
       DECLARE_NODE_API_PROPERTY("invalidObjectAsBuffer", invalidObjectAsBuffer),
+      DECLARE_NODE_API_PROPERTY("bufferFromArrayBuffer", bufferFromArrayBuffer),
```

Comment:
The new test function needs to be invoked in https://github.com/nodejs/node/blob/main/test/node-api/test_buffer/test.js in order to run in actual test.

Reply:
thanks, ı added in test.js